pb/badgerpb2.proto

```diff
@@ -19,7 +19,7 @@ syntax = "proto3";
 
 package badgerpb2;
```

Comment:
@aman-bansal you will have to change this to v3. see https://github.com/dgraph-io/badger/pull/1293